src/wallet/load.cpp

```diff
@@ -53,10 +54,10 @@ bool VerifyWallets(interfaces::Chain& chain, const std::vector<std::string>& wal
         }
 
         std::string error_string;
-        std::string warning_string;
+        std::vector<std::string> warning_string;
```

Comment:
µ-nit: use `warning_strings` (plural) to be consistent with other changes.

Reply:
Will do when there is other stuff to change